Intermittent selenium test failures due to Zendesk

> AssertionError: [{'level': 'SEVERE', 'message': 'https://odl.zendesk.com/embeddable/config - Failed to load resource: the server responded with a status of 504 (Gateway Time-out)', 'timestamp': 1488902695097}, {'level': 'SEVERE', 'message': 'https://odl.zendesk.com/embeddable/config - Failed to load resource: the server responded with a status of 504 (Gateway Time-out)', 'timestamp': 1488902695097}, {'level': 'SEVERE', 'message': "http://172.18.0.6:8286/learners/ - XMLHttpRequest cannot load https://odl.zendesk.com/embeddable/config. No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://172.18.0.6:8286' is therefore not allowed access. The response had HTTP status code 504.", 'timestamp': 1488902695097}, {'level': 'SEVERE', 'message': 'http://172.18.0.6:8286/static/bundles/zendesk_widget-634b145b247da81e80f8.js 0:3068 "couldn\'t find Zendesk iframe: ipm"', 'timestamp': 1488902695097}, {'level': 'SEVERE', 'message': 'http://172.18.0.6:8286/static/bundles/zendesk_widget-634b145b247da81e80f8.js 0:3068 "couldn\'t find Zendesk iframe: ticketSubmissionForm"', 'timestamp': 1488902695097}, {'level': 'SEVERE', 'message': 'http://172.18.0.6:8286/static/bundles/zendesk_widget-634b145b247da81e80f8.js 0:3068 "couldn\'t find Zendesk iframe: launcher"', 'timestamp': 1488902695097}, {'level': 'SEVERE', 'message': 'http://172.18.0.6:8286/static/bundles/zendesk_widget-634b145b247da81e80f8.js 0:3068 "couldn\'t find Zendesk iframe: nps"', 'timestamp': 1488902695097}]

--- a/selenium_tests/base.py
+++ b/selenium_tests/base.py
@@ -264,6 +264,8 @@ FROM pg_stat_activity WHERE pid <> pg_backend_pid()""")
                 continue
             if "'webkitURL' is deprecated. Please use 'URL' instead" in message:
                 continue
+            if "zendesk" in message.lower():
+                continue
 
             # warnings (e.g. deprecations) should not fail the tests
             if entry['level'] in ["WARNING"]:

